types/mui-datatables/index.d.ts

```diff
@@ -200,7 +210,7 @@ export interface MUIDataTableIsRowCheck {
         {
             index: number;
             dataIndex: number;
-        }
+        },
```

Comment:
```suggestion
        }
```

No need for a trailing comma here